### ISSUE TITLE
Only the first Resource Duplicate is found - E0000

### ISSUE THREAD

**Issue by asantos-fuze** [aws-cloudformation/cfn-python-lint#1868]
*cfn-lint version: (`cfn-lint 0.44.3`)*

*Description of issue.*

I get the error `E0000 Duplicate resource found "DevRouteTableRoute2" (line 18)` when I try to validate a CFn template with duplicated resources. I should get has many errors has the number of duplicated resources on the template. The template had more than one duplication, but the linter only showed the 1st.

Please provide as much information as possible:
* Test Template:
```yaml
---
AWSTemplateFormatVersion: "2010-09-09"

Resources:

  DevRouteTableRoute1:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      Blackhole: true
      DestinationCidrBlock: 10.1.1.0/24
      TransitGatewayRouteTableId: tgw-rtb-d65d85280f2c9e51c
  DevRouteTableRoute2:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      DestinationCidrBlock: 10.1.2.0/24
      TransitGatewayAttachmentId: tgw-attach-9bdf1fdfb44d9c7d7
      TransitGatewayRouteTableId: tgw-rtb-d65d85280f2c9e51c
  DevRouteTableRoute2:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      DestinationCidrBlock: 10.1.3.0/24
      TransitGatewayAttachmentId: tgw-attach-b16ea962254d4373e
      TransitGatewayRouteTableId: tgw-rtb-d65d85280f2c9e51c
  DevRouteTableRoute4:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      DestinationCidrBlock: 10.1.4.0/24
      TransitGatewayAttachmentId: tgw-attach-b16ea962254d4373e
      TransitGatewayRouteTableId: tgw-rtb-d65d85280f2c9e51c

  ProdRouteTableRoute1:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      Blackhole: true
      DestinationCidrBlock: 10.1.1.0/24
      TransitGatewayRouteTableId: tgw-rtb-5e9891f700a3af932
  ProdRouteTableRoute2:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      DestinationCidrBlock: 10.1.2.0/24
      TransitGatewayAttachmentId: tgw-attach-9bdf1fdfb44d9c7d7
      TransitGatewayRouteTableId: tgw-rtb-5e9891f700a3af932
  ProdRouteTableRoute2:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      DestinationCidrBlock: 10.1.3.0/24
      TransitGatewayAttachmentId: tgw-attach-b16ea962254d4373e
      TransitGatewayRouteTableId: tgw-rtb-5e9891f700a3af932
  ProdRouteTableRoute4:
    Type: AWS::EC2::TransitGatewayRoute
    Properties:
      DestinationCidrBlock: 10.1.4.0/24
      TransitGatewayAttachmentId: tgw-attach-b16ea962254d4373e
      TransitGatewayRouteTableId: tgw-rtb-5e9891f700a3af932
```

* Validation Output
```
cfn-lint -t template.yml
E0000 Duplicate resource found "DevRouteTableRoute2" (line 18)
template.yml:18:3
```

* Expected Output
```
cfn-lint -t template.yml
E0000 Duplicate resource found "DevRouteTableRoute2" (line 18)
template.yml:18:3

E0000 Duplicate resource found "ProdRouteTableRoute2" (line 43)
template.yml:43:3
```

I've run manually cfn-lint -u but the validation on cfn-lint only shows on duplicate instead of the two that exist in the template.
